operators/pkg/controller/elasticsearch/initcontainer/script.go

```diff
@@ -121,6 +123,19 @@ var scriptTemplate = template.Must(template.New("").Parse(
 	{{end}}
 	echo "chown duration: $(duration $chown_start) sec."
 
+	######################
+	#  Wait for certs    #
+	######################
+
+	# wait for the transport certificates to show up
+	echo "waiting for the transport certificates"
+	wait_start=$(date +%s)
+	while [ ! -f {{ .TransportCertificatesKeyPath }} ]
+	do
+	  sleep 1
```

Comment:
Maybe we can make it even shorter? `sleep 0.2`?

Reply:
We certainly can. I don't have particularly strong opinions on the specific number, so let's go with 0.2 for now then.